src/test/kotlin/com/beust/klaxon/Iso8601Utils.kt

```diff
@@ -0,0 +1,231 @@
+package com.beust.klaxon
+
+import java.util.Calendar
+import java.util.Date
+import java.util.GregorianCalendar
+import java.util.TimeZone
+
+/**
+ * Jackson’s date formatter, pruned to Moshi's needs. Forked from this file:
```

Comment:
Can you explain what you mean by "pruned to Moshi's needs"? What does Moshi have to do with this?